tests/prague/eip2537_bls_12_381_precompiles/conftest.py

```diff
@@ -11,9 +11,22 @@
 
 
 @pytest.fixture
-def precompile_gas(precompile_address: int, input_data: bytes) -> int:
+def vector_gas_value() -> int | None:
+    """Gas value from the test vector."""
+    return None
+
+
+@pytest.fixture
+def precompile_gas(
+    precompile_address: int, input_data: bytes, vector_gas_value: int | None
+) -> int:
     """Gas cost for the precompile."""
-    return GAS_CALCULATION_FUNCTION_MAP[precompile_address](len(input_data))
+    calculated_gas = GAS_CALCULATION_FUNCTION_MAP[precompile_address](len(input_data))
+    if vector_gas_value is not None:
```

Comment:
Just to check: are the vector gas params imported here correctly?

Reply:
It is, clarified here: https://github.com/ethereum/execution-spec-tests/pull/1063#discussion_r1916915855